<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>747</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">BendingPriceCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.011, y1:0.852455, x2:0.96, y2:0.085, stop:0 rgba(112, 2, 117, 255), stop:1 rgba(128, 16, 255, 255));</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="bending_price_calc_window">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="company_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(0, 255, 162);</string>
    </property>
    <property name="locale">
     <locale language="Ukrainian" country="Ukraine"/>
    </property>
    <property name="text">
     <string>Компанія</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="company_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(0, 255, 162);
	color: rgb(202, 8, 255);
}

QLineEdit:hover {
	border: 3px solid rgb(2, 35, 255);
}
QLineEdit:focus {

	border: 3px solid rgb(225, 255, 249);
}</string>
    </property>
    <property name="text">
     <string>Nttg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="type_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(0, 255, 162);</string>
    </property>
    <property name="text">
     <string>Tип</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="type_value">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(237, 255, 152);
	color: rgb(202, 8, 255);
}
QComboBox:hover {
	border: 3px solid rgb(2, 35, 255);
}
QComboBox:focus {

	border: 3px solid rgb(225, 255, 249);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="item_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(0, 255, 162);</string>
    </property>
    <property name="text">
     <string>Виріб</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="item_value">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QComboBox{&lt;/p&gt;&lt;p&gt;	background-color: rgb(237, 255, 152);&lt;/p&gt;&lt;p&gt;	color: rgb(202, 8, 255);&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;p&gt;QComboBoxt:hover {&lt;/p&gt;&lt;p&gt;	border: 3px solid rgb(2, 35, 255);&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;p&gt;QComboBox:focus {&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;	border: 3px solid rgb(225, 255, 249);&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(237, 255, 152);
	color: rgb(202, 8, 255);
}
QComboBox:hover {
	border: 3px solid rgb(2, 35, 255);
}
QComboBox:focus {

	border: 3px solid rgb(225, 255, 249);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="number_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(0, 255, 162);</string>
    </property>
    <property name="text">
     <string>Номер</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="number_value">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(237, 255, 152);
	color: rgb(202, 8, 255);
}
QComboBox:hover {
	border: 3px solid rgb(2, 35, 255);
}
QComboBox:focus {

	border: 3px solid rgb(225, 255, 249);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="length_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(0, 255, 162);</string>
    </property>
    <property name="text">
     <string>Довжина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="length_value">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(237, 255, 152);
	color: rgb(202, 8, 255);
}
QComboBox:hover {
	border: 3px solid rgb(2, 35, 255);
}
QComboBox:focus {

	border: 3px solid rgb(225, 255, 249);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantity_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(0, 255, 162);</string>
    </property>
    <property name="text">
     <string>Кількість</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantity_value">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
	background-color: rgb(237, 255, 152);
	color: rgb(202, 8, 255);
}
QSpinBox:hover {
	border: 3px solid rgb(2, 35, 255);
}
QSpinBox:focus {

	border: 3px solid rgb(225, 255, 249);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_item_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color:white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(102, 158, 255, 255), stop:0.55 rgba(71, 61, 235, 255), stop:0.98 rgba(27, 2, 212, 255), stop:1 rgba(0, 0, 0, 0));
border-radius: 20px;
}

QPushButton:hover {
    background-color:  rgb(48, 221, 255);
}</string>
    </property>
    <property name="text">
     <string>Додати виріб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 102, 102, 255), stop:0.5625 rgba(194, 80, 21, 255), stop:0.98 rgba(196, 4, 56, 255), stop:1 rgba(0, 0, 0, 0));
color:white;
border-radius: 20px;
}

QPushButton:hover {
    background-color:  rgb(48, 221, 255);
}</string>
    </property>
    <property name="text">
     <string>Все с початку</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>481</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(237, 255, 152);
color: rgb(199, 55, 255);</string>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotLine</enum>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>test</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Код</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Назва</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кількість</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="weight_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Вага складає</string>
    </property>
   </widget>
   <widget class="QLabel" name="weight_value">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>0000.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="kg_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>кг</string>
    </property>
   </widget>
   <widget class="QLabel" name="lenght_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 102, 0);</string>
    </property>
    <property name="text">
     <string>Максимальна довжина одного виробу</string>
    </property>
   </widget>
   <widget class="QLabel" name="lenght_value">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>620</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 102, 0);</string>
    </property>
    <property name="text">
     <string>000.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="cm_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>35</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 102, 0);</string>
    </property>
    <property name="text">
     <string>см</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xlsx_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(84, 198, 11, 255), stop:0.369318 rgba(23, 104, 10, 255), stop:0.98 rgba(13, 212, 2, 255), stop:1 rgba(0, 0, 0, 0));
color:white;
border-radius: 20px;
}

QPushButton:hover {
    background-color:  rgb(48, 221, 255);
}</string>
    </property>
    <property name="text">
     <string>Створити xlsx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_table_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>660</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(243, 33, 61, 255), stop:0.55 rgba(235, 61, 61, 255), stop:0.98 rgba(134, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color:white;
border-radius: 20px;
}

QPushButton:hover {
    background-color:  rgb(48, 221, 255);
}</string>
    </property>
    <property name="text">
     <string>Скинути таблицю</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
